### ISSUE TITLE
Accessibility: Pagination Control

### ISSUE THREAD

**Issue by EricDunaway** [quasarframework/quasar#12637]
### What happened?

The pagination control lacks accessible names for the buttons. When a button doesn't have an accessible name, screen readers announce it as "button", making it unusable for users who rely on screen readers.

### What did you expect to happen?

I expect the pagination control to provide accessible names for the buttons or provide a way to set them through props or other means.

### Reproduction URL

https://quasar.dev/vue-components/pagination

### How to reproduce?

1. Goto https://web.dev/measure/
2. Audit `https://quasar.dev/vue-components/pagination`
3. Click the `Accessibility` tab of the report
4. Read the section labeled `Buttons do not have an accessible name`

### Flavour

Vite Plugin (@quasar/vite-plugin)

### Areas

Components (quasar)

### Platforms/Browsers

Firefox, Chrome, Safari, Microsoft Edge, iOS, Android

**Comment by yusufkandemir** [comment]
Fixed in https://github.com/quasarframework/quasar/commit/3913d4ea5fc7a2273c08c2045559efc78f5eebff, available since Quasar v2.10.0.
